architecture/core_concepts/pods_and_services.adoc

```diff
@@ -160,6 +161,57 @@ link:https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-gui
 has details about the functionality and purpose of pods.
 ====
 
+[[init-containers]]
```

Comment:
Nice catch. I wasn't aware of this section in the Containers and Images topic. I changed the link in that section from the Kubernetes doc to this one topic.